### PR TITLE
Trend micro vision one v3

### PR DESCRIPTION
**Pull request by shaqnawe**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Updated V3 app description in YML and updated docker image to: demisto/python3:3.10.9.46807. Also updated UTs per fix mentioned here: https://github.com/demisto/content/pull/24273/files#

## Screenshots
N/A

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/TrendMicroVisionOne/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Trend Micro Vision One",
     "description": "Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response(XDR) capabilities that collect and automatically correlate data across multiple security layers\u2014email, endpoints, servers, cloud workloads, and networks\u2014Trend Micro Vision One prevents the majority of attacks with automated protection.",
     "support": "partner",
-    "currentVersion": "1.3.0",
+    "currentVersion": "2.0.2",
```
Comment (kgal-pan) [inline Packs/TrendMicroVisionOne/pack_metadata.json:5]: How did we jump from 1.3.0 to 2.0.2? In `master` we're in version `2.0.1`. Let's fix this.

File: Packs/TrendMicroVisionOne/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### New: Trend Micro Vision One V3.
+- Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response (XDR) capabilities that collect and automatically correlate data across multiple security layers—email, endpoints, servers, cloud workloads, and networks—Trend Micro Vision One prevents the majority of attacks with automated protection. (Available from Cortex XSOAR 6.5.0).
+
+#### Mappers
+##### New: Trend Micro Vision One V3 XDR - Incoming Mapper
+- Added a new mapper for V3 (Available from Cortex XSOAR 6.5.0).
```
Comment (kgal-pan) [inline Packs/TrendMicroVisionOne/ReleaseNotes/2_0_0.md:8]: This should be in `2_0_2.md` as it was added in this PR.

**Comment by shaqnawe** [comment]
Working branch was not updated, will update branch and create new PR.

**Comment by kgal-pan** [comment]
new PR was https://github.com/demisto/content/pull/24442.

**Pull request merged**